tests/timeseries/test_quantities.py

```diff
@@ -45,6 +48,48 @@ class Fz2(
     """Frequency quantity with broad exponent unit map."""
 
 
+_CtorType = Callable[[float], Quantity]
+
+# Thi is the current number of subclasses. This probably will get outdated, but it will
+# provide at least some safety against something going really wrong and end up testing
+# an empty list. With this we should at least make sure we are not testint less classes
+# than before. We don't get the actual number using len(_QUANTITY_SUBCLASSES) because it
+# would defeat the purpose of the test.
```

Comment:
```suggestion
# This is the current number of subclasses. This probably will get outdated, but it will
# provide at least some safety against something going really wrong and end up testing
# an empty list. With this we should at least make sure we are not testing less classes
# than before. We don't get the actual number using len(_QUANTITY_SUBCLASSES) because it
# would defeat the purpose of the test.
```